src/utilities/flags.ts

```diff
@@ -47,34 +51,40 @@ export default class Flags {
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export const flagDefinitions: Record<string, any> = {
   [Flags.CLEAN]: OclifFlags.boolean({
+    char: 'c',
     default: false,
-    description: 'Clean the theme directory before copying components'
+    description: 'Remove unreferenced translations from theme locale files'
   }),
 
   [Flags.COLLECTION_NAME]: OclifFlags.string({
     char: 'n',
-    description: 'name of the component collection',
+    description: 'Name of the component collection',
   }),
 
   [Flags.COLLECTION_VERSION]: OclifFlags.string({
     char: 'v',
-    description: 'version of the component collection',
+    description: 'Version of the component collection',
   }),
 
   [Flags.ENVIRONMENT]: OclifFlags.string({
     description: 'The environment to apply to the current command.',
   }),
 
+  [Flags.FORMAT]: OclifFlags.boolean({
+    default: false,
+    description: 'Format locale files by sorting keys alphabetically',
+  }),
```

Comment:
If this defaults to false, in what order are new locales keys added?

Reply:
Good question. With format as false, the order corresponds to the sequence in which they appear in the Liquid files.